src/components/DocumentationWindow.vue

```diff
@@ -0,0 +1,65 @@
+                <h4> Where do the annotations come from? </h4>
+                    <p> The categories and terms shown are derived from the concepts associated with datasets. Datasets are annotated to capture the most salient features of the experimental setting and design (e.g., the treatments applied to certain parts of organisms at specific life stages to study particular diseases).</p>
+                <br>
+                <h3>Buttons</h3>
+                    <p> When filtering selections have been made, a summary of the filter categories used will be displayed in the top bar of GemBrow. To view a more detailed description of the selections, click this text <v-btn plain style="text-transform: none;" small> Filters applied: <v-icon>mdi-chevron-down</v-icon></v-btn>. Click anywhere to exit the filter description menu.</p>
+
+                    <p> To download a .tsv file with the data from current selections, click this <v-btn plain small><v-icon>mdi-download</v-icon></v-btn>icon. Click <v-btn plain small><v-icon>mdi-cancel</v-icon></v-btn> icon to cancel downloads. </p>
+
+                    <p> Some users may analyse Gemma data in other applications. Open this window <v-btn plain small style="text-transform: none;">Dataset Download Code <v-icon>mdi-chevron-up</v-icon></v-btn>,  select a tab, and use the <v-btn plain small><v-icon>mdi-clipboard-outline</v-icon></v-btn> icon to copy code that downloads the current filtered dataset with python, R, curl, or HTTP. Click anywhere to exit the menu.</p>
+            </v-card-text>
+        </v-card>
+    </v-dialog>
+</template>
+<script>
+import { mapState } from "vuex";
+import { baseUrl, marked } from "@/config/gemma";
+
+export default {
+  props: {
+    value: Boolean
+  },
+  data() {
+    return {
+      baseUrl
+    };
+  },
+  computed: mapState({
+    root: state => state.api.root?.data || { externalDatabases: [] }
+  }),
+  methods: {
+    m2h(m) {
+      return marked.parseInline(m);
+    }
+  },
+  created() {
+    this.$store.dispatch("api/getRoot").catch(e => {
+      console.error("Failed to retrieve the root endpoint: " + e.message + ".", e);
+    });
+  }
```

Comment:
You don't need that, this is only used by the About window to retrieve external databases status.